Cookbook/Technical-Documents/How to use Slack?.md

```diff
@@ -0,0 +1,146 @@
+To understand the status of a message we tag the messages following:
+
+- In discussion 💬
+- No outcome ❎
+- With outcome ✅
+- Announcement ⚠️
+
+To be easier to identify the topic of the message we use:
+ `[<topic of the message>] message`
+
+Example: 
+
+```
+[Peakon] Please make sure you feel the Peakon survey frequently and give as much feedback as you can. Please do not ever think your feedback is not relevant enough to write it down. I can guarantee you every feedback is important for us and it will always help us get better at what we do.
+Your help is very much appreciated!
+Thank you :thanks:
+```
+
+### #ios-questions
+
+A private channel where the iOS engineers can post any technical question and it will be answered by anyone in the iOS Team.
+The questions should be a message in a channel and the reply should be withing the message's thread.
+Ideally follow the stackoverflow approach and state what you tried.
+
+The question should follow the format:
+`[<topic of the question>] question`
+
+Example:
+```
+[RAF] What is an example with the newest style for a page where a child triggers some events in a parent? What is best to inject, and from where?
+```
+
+### #ios-build
+
+A public channel where we usually trigger CI workflows like Testfligh/AppCenter builds or UI Automation Tests.
+The UI Automation Tests run every night and the results are published in this channel by the bot.
+To check how to trigger the workflows check [Slack CI Integration](/SlackCIIntegration.md)
+
+## How to manage notifications?
+
+Slack notifications are great to make sure we don't miss anything important. However, when they are very frequent they can disturb our work and our productivity.
+
+There are a few tips to try to reduce the number of notifications to the ones that really matter to you:
+
+### Global notification preferences
+
+To reduce the overall number of notifications we can configure the notifications on Slack doing the following:
+
+1. Go to Slack > Preferences > Notifications
+2. Configure your notifications with:
+
+<img src="Assets/onboarding/slack-notification-preferences.png">
+
+3. Go to `My keywords` and define the keywords to be used to trigger notifications. These can be your nicknames and variations of your name.
+
+4. In the `Do Not Disturb` you can set the time interval you don't want to receive any Slack notification. Ideally this should enabled during your non-working hours.
+
+On the day to day basis, if you want some quiet time, you can mute notifications temporarly by selecting the notifications icon next to your Slack organization name followed by selecting one of the options.
+
+<img src="Assets/onboarding/slack-notification-muted.png">
+
+### Leave the channel
+
+When you leave a channel you can no longer open it unless you join it again. Also, when someone shares a thread from that channel with you, you will have to join the channel to be able to read it.
```

Comment:
You can open public channels, but not participate without joining. Only private channel's can't be re-joined since you can't even see them

```suggestion
When you leave a channel, you can see its messages, but can't participate unless you join it again. When someone shares a thread from that channel, you will be able to read it, but again you'll have to join the channel to interact with it.
```